main.py

```diff
@@ -0,0 +1,26 @@
+from flask import Flask, redirect, url_for, render_template
+
+app = Flask(__name__)
+
+# this url route displays some html tags on the home page
+@app.route("/")
+def home():
+    return "<h1>Image Analysis System</h1> <p>Hello world!</p>"
+
+# this url route will take the input passed in and display it
+@app.route("/<name>/")
+def testURLInput(name):
+    return f"Hello {name}!"
+
+# this url route will redirect to the home page
+@app.route("/admin/")
+def redirectWithoutParam():
+    return redirect(url_for("home"))
+
+# this url route will redirect to the /name page with a specific param
+@app.route("/admin_param/")
+def redirectWithParam():
+    return redirect(url_for("testURLInput", name="Dr. Huda"))
```

Comment:
These are some test routes that do some example actions:
`/`:  This is the home page, just going to the dev URL printed out in the server takes you to this page
`/<name>`: This page will take in a param (*name*) from the URL and print it out on the screen
`/admin`: This page demonstrates rerouting. If you go to this page, you will be rerouted to `home`
`/admin_param`: This page demonstrates rerouting with a parameter, you will be rerouted to `/<name>` with the specified parameter

These are all temporary, and just demo endpoints to play around with as we get comfortable with how `Flask` works!